docs/UsingTheIOSubsystem.md

```diff
@@ -0,0 +1,615 @@
+| setsockopt        |                                                          |
+| shutdown          |                                                          |
+| socket            |                                                          |
+| socketpair        |                                                          |
+|                   | <img width="1000">                                       |
+
+**<select.h>**
+-------------
+
+For the **<select.h>** header, the I/O subsystem adds support for the following
+functions.
+
+| Function          | Limitations                                              |
+| :---              | :---                                                     |
+| FD_CLR            |                                                          |
+| FD_ISSET          |                                                          |
+| FD_SET            |                                                          |
+| FD_ZERO           |                                                          |
+| select            |                                                          |
+|                   | <img width="1000">                                       |
+
+**<dirent.h>**
+-------------
+
+For the **<dirent.h>** header, the I/O subsystem adds support for the following
+functions.
+
+| Function          | Limitations                                              |
+| :---              | :---                                                     |
+| closedir          |                                                          |
+| opendir           |                                                          |
+| readdir           |                                                          |
+| readdir_r         |                                                          |
+| rewinddir         |                                                          |
+| telldir           |                                                          |
+|                   | <img width="1000">                                       |
+
+**<poll.h>**
+-------------
+
+For the **<poll.h>** header, the I/O subsystem adds support for the following
+functions.
+
+| Function          | Limitations                                              |
+| :---              | :---                                                     |
+| poll              |                                                          |
+|                   | <img width="1000">                                       |
+
+**<poll.h>**
+-------------
+
+For the **<poll.h>** header, the I/O subsystem adds support for the following
+functions.
+
+| Function          | Limitations                                              |
+| :---              | :---                                                     |
+| poll              |                                                          |
+|                   | <img width="1000">                                       |
+
```

Comment:
Delete the second "poll.h" section as this is a repetition (accidental, I suppose)

Reply:
Thanks for catching that. Resolved.